src/com/ss/ita/util/implementation/ConsoleScanner.java

```diff
@@ -81,20 +81,20 @@ public double readDouble() {
     public double[] readDoubleArray() {
         List<Double> doubles = new ArrayList<>();
         while (true) {
-            Double number = scanner.nextDouble();
-            if (number != null) {
-                doubles.add(number);
-            } else {
-                double[] doubleArray=new double[doubles.size()];
-                for (int i=0;i<doubles.size();i++){
-                    doubleArray[i]=doubles.get(i);
+            if (scanner.hasNextInt()) {
```

Comment:
Please, fix if statement, "hasNextInt" is incorrect

Reply:
fixed